tests/suite_test.go

```diff
@@ -113,11 +104,8 @@ func init() {
 	flag.StringVar(&DSPANamespace, "DSPANamespace", DefaultDSPANamespace, "The namespace to deploy DSPA.")
 
 	flag.DurationVar(&DeployTimeout, "DeployTimeout", DefaultDeployTimeout, "Seconds to wait for deployments. Consider increasing this on resource starved environments.")
-	DeployTimeout *= time.Second
 	flag.DurationVar(&PollInterval, "PollInterval", DefaultPollInterval, "Seconds to wait before retrying fetches to the api server.")
-	PollInterval *= time.Second
 	flag.DurationVar(&DeleteTimeout, "DeleteTimeout", DefaultDeleteTimeout, "Seconds to wait for deployment deletions. Consider increasing this on resource starved environments.")
```

Comment:
you set the default time here: https://github.com/opendatahub-io/data-science-pipelines-operator/pull/589/files#diff-96e66e1ccb0083ac5a2b7fc37cb002803d242ea67c1981a5d09ef8ee2668354cR61-R63 

but it looks like if the user specifies a time, we don't convert to the proper time.duration, did you confirm this? it makes sense to me to keep this here, but remove it from the const()

Reply:
Done, kept it here and removed from const()